static/javascript/index.js

```diff
@@ -1,5 +1,112 @@
+///// for access to environmental variable /////
+var app_id
+
+// This is called with the results from from FB.getLoginStatus().
+function statusChangeCallback(response) {
+    console.log('statusChangeCallback');
+    console.log(response);
+    // The response object is returned with a status field that lets the
+    // app know the current login status of the person.
+    // Full docs on the response object can be found in the documentation
+    // for FB.getLoginStatus().
+    if (response.status === 'connected') {
+      // Logged into your app and Facebook
+        
+      testAPI(response);
+    } else if (response.status === 'not_authorized') {
+      // The person is logged into Facebook, but not your app.
+      document.getElementById('status').innerHTML = 'Please log ' +
+        'into this app.';
+    } else {
+      // The person is not logged into Facebook, so we're not sure if
+      // they are logged into this app or not.
+      
+      FB.getLoginStatus(function(response) {
+        $.ajax({
+            url: '/log_out',
+            type: 'DELETE',
+            success: sign_out
+        });
+            console.log("In getLoginStatus");
+
+        });
```

Comment:
I'm talking about `FB.getLoginStatus()`.  click "view changes" to see more context.